crates/algod_client/README.md

```diff
@@ -0,0 +1,237 @@
+# Algod REST API.
+
+API endpoint for algod operations.
+
+**Version:** 0.0.1
+**Contact:** contact@algorand.com
+
+This Rust crate provides a client library for the Algod REST API. API.
+
+## Installation
+
+Add this to your `Cargo.toml`:
+
+```toml
+[dependencies]
+algod_client = "0.0.1"
+```
+
+## Usage
+
+```rust
+use algod_client::apis::{configuration, default_api};
+
+#[tokio::main]
+async fn main() -> Result<(), Box<dyn std::error::Error>> {
+    let configuration = configuration::Configuration::new();
+    
+    // Example API call
```

Comment:
Not a great example

Reply:
Good point, missed this one as it was made before the tests were actually written, changed to include some more expanded samples